script.js

```diff
@@ -35,18 +45,15 @@ function createEpisodeContainer() {
 
   return container;
 }
-
 // Function to pad numbers with leading zeros
 const pad = (n) => {
   if (typeof n !== "number" || isNaN(n)) return "00";
   return n.toString().padStart(2, "0");
 };
-
 // Template for episode cards
 const template = document.getElementById("episode-card-template");
-
 // Function to create the episode card
-function createEpisodeCard(episode) {
+function createEpisodeCard(episode) { // It will be this the render function?
```

Comment:
@Emilianouz The comment appears to be leftover development notes and should be removed for cleaner code.